servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingToStreamingService.java

```diff
@@ -88,8 +86,9 @@ protected void handleSubscribe(final Subscriber<? super StreamingHttpResponse> s
                             // Content-Length header field can provide the anticipated size.
                             // https://tools.ietf.org/html/rfc7230#section-3.3.2
                             HttpHeaders headers = metaData.headers();
-                            boolean addTrailers = isTransferEncodingChunked(headers);
-                            if (!addTrailers && !HTTP_1_0.equals(metaData.version()) && !hasContentLength(headers) &&
+                            boolean addTrailers = metaData.version().major() > 1 || isTransferEncodingChunked(headers);
+                            if (!addTrailers && metaData.version().major() == 1 && metaData.version().minor() > 0 &&
+                                    !hasContentLength(headers) &&
```

Comment:
After #1311 we will be able to share `netty/HeaderUtils` and reuse these utilities

Reply:
it would be nice to consolidate these trailers checks.